EXILED_Events/Patches/GameConsoleQueryOverride.cs

```diff
@@ -14,7 +14,7 @@ public static bool Prefix(RemoteAdmin.QueryProcessor __instance, ref string quer
 					return true;
 				
 				Events.InvokeConsoleCommand(__instance.gameObject, query, encrypted, out string returnMessage, out string color);
-				if (color == null)
+				if (!string.IsNullOrEmpty(color))

```

Comment:
this should be if (string.IsNullOrEmpty(color)) not if (!...), otherwise it will always change the color to white if the color is *NOT* null/empty, and hit an exception if it is.

Reply:
Oh well, sure, my bad. Fixed.